cmd/get.go

```diff
@@ -166,10 +169,64 @@ var getToken = &cobra.Command{
 	},
 }
 
+var getOrganizationCmd = &cobra.Command{
+	Use:     "organization",
+	Aliases: []string{"o"},
+	Short:   "Get details about an organization",
+	PreRunE: func(_ *cobra.Command, _ []string) error {
+		return validateTokenE(cmdLagoon)
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		debug, err := cmd.Flags().GetBool("debug")
+		if err != nil {
+			return err
+		}
+		organizationName, err := cmd.Flags().GetString("organization")
+		requiredInputCheck("Organization name", organizationName)
+		if err != nil {
+			return err
+		}
```

Comment:
This is a bit funky, I'd change it for the following as long as `requiredInputCheck` is set to return an error.
```
[		organizationName, err := cmd.Flags().GetString("organization")
		if err != nil {
			return err
		}]
		if err := requiredInputCheck("Organization name", organizationName); err != nil {
			return err
		}
```

The `exit(1)` is an old CLI command hangover, and need to stop using that

Reply:
Same for any other occurrence of this.

The main reasons are that you capture the errors in the order they could be received. At the moment you're throwing something into the requiredinputcheck function before there is the possibility to handle the error it could have thrown, and that could lead to unknown error/panic